config/internal/manager/crd/bases/catalog.kubecarrier.io_catalogentrysets.yaml

```diff
@@ -82,6 +82,13 @@ spec:
                             type: object
                           minItems: 1
                           type: array
+                        patch:
```

Comment:
How you can change this without adding `patch` field in the `CatalogEntrySet.Spec` in the catalogentryset_types.go?
And also for catalogentry_types.go

Reply:
Also, don't forget to pass the `patch` field from `CatalogEntrySet` to `CatalogEntry` and then `DerivedCR`, and add tests for that, otherwise, it can lead bugs like provider specifies the `Patch` in the `CatalogEntrySet` but can not see that in the `DerivedCR` object.

Reply:
They have added automatically since those objects are reusing the same struct and passing derived spec data down to the DerivedCR. Thus everything works as expected.